pkgs/by-name/ri/rip2/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rip2";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "MilesCranmer";
+    repo = "rip2";
+    rev = "v${version}";
+    hash = "sha256-9leLWfPilDQHzQRzTUjAFt9olTPEL4GcQgYFWZu3dug=";
+  };
+
+  cargoHash = "sha256-l6rbeiyIsr1csBcp+428TpQYSs9RvfJutGoL/wtSGR8=";
+
+  # TODO: Unsure why this test fails, but not a major issue so
+  #       skipping for now.
+  checkFlags = [ "--skip=test_filetypes::file_type_3___fifo__" ];
+
+  meta = {
+    description = "Safe and ergonomic alternative to rm";
+    homepage = "https://github.com/MilesCranmer/rip2";
+    license = lib.licenses.gpl3Plus;
+    maintainers = with lib.maintainers; [ milescranmer ];
+    mainProgram = "rip";
+    platforms = lib.platforms.all;
```

Comment:
```suggestion
```

`rustPlatform.buildRustPackage` carries its own supported platforms, so if it's not needed to restrict then we don't have to add anything here